bin/parse_ectyperQC.py

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/env python3
```

Comment:
As future work I do wonder if any of these scripts share common code that could be moved to a package they all import.